packages/runtime/container-runtime/src/summaryFormat.ts

```diff
@@ -77,16 +77,21 @@ export function hasIsolatedChannels(attributes: ReadFluidDataStoreAttributes): b
 export type GCVersion = number;
 export interface IContainerRuntimeMetadata extends ICreateContainerMetadata {
     readonly summaryFormatVersion: 1;
-    /** The last message processed at the time of summary. Only primitive propertiy types are added to the summary. */
+    /** The last message processed at the time of summary. Only primitive property types are added to the summary. */
     readonly message: ISummaryMetadataMessage | undefined;
     /** True if channels are not isolated in .channels subtrees, otherwise isolated. */
     readonly disableIsolatedChannels?: true;
     /** 0 to disable GC, > 0 to enable GC, undefined defaults to disabled. */
     readonly gcFeature?: GCVersion;
-    /** Counter of the last summary happened, increments every time we summarize */
-    readonly summaryCount?: number;
+    /** Represents the summary number for a document. Incremented on summary throughout a document's lifetime. */
+    readonly summaryNumber?: number;
     /** If this is present, the session for this container will expire after this time and the container will close */
     readonly sessionExpiryTimeoutMs?: number;
+    /**
+     * @deprecated - User summaryNumber instead.
+     * Counter of the last summary happened, increments every time we summarize
+     * */
+    readonly summaryCount?: number;
```

Comment:
>summaryCount [](http://example.com/codeflow?start=13&length=12)

out of curiosity, do we need to document this deprecation ?

Reply:
Are you talking about adding it to breaking.md? It's not part of public API surface so we don't need to do that. However, we could create an issue to remove this later.